keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -200,7 +186,7 @@ From here, a group can search the open items labeled with
 A nice example of a written SIG workflow is the [grooming document from
 SIG Cluster Lifecycle](https://git.k8s.io/community/sig-cluster-lifecycle/grooming.md).
 
-#### Remove or rename unused triage/** labels
+### Remove or rename unused triage/** labels
```

Comment:
GitHub doesn't allow me to comment there...in the line where we mention this:

```
`triage/needs-information` --> `lifecycle/needs-information`: Issue requires more information (from submitter or SIG) in order to work on it
```

As per the current behavior of lifecycle labels, we allow only one `lifecycle/*` label on the issue. For example, if someone comments with `/lifecycle frozen`, we remove the `lifecycle/rotten` label.

Since the KEP would now be "implementable", I think we should explicitly call out that `lifecycle/needs-information` would NOT be auto-removed if another `lifecycle/*` label is applied.

Reply:
hmm...I don't think we want to disrupt that. I know the goal is to make `needs-triage` and `/triage accepted` a type of boolean, but `triage` does seem like the best home for `needs-information`. 

I know its not necessarily ideal, but what about leaving it under `triage`, and gauge it's usage for a release or 2 after we put this new workflow into place?

cc @justaugustus @thockin

Reply:
> I know its not necessarily ideal, but what about leaving it under triage, and gauge it's usage for a release or 2 after we put this new workflow into place?

+1